extensions/blocks/conversation/edit.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import { filter, map } from 'lodash';
```

Comment:
Nitpick and non-blocking: Same as above here, these methods are supported by native arrays in JS so we can decrease the bundle size a little bit by using them.